lib/sphinx_rtm/messages.ex

```diff
@@ -29,8 +29,7 @@ defmodule SphinxRtm.Messages do
             |> Map.put(:text, content)
             |> save_question()
 
-            # It will change for {:reply, "You're question is saved"} by post_ephemeral
```

Comment:
I think we can change` {:reply, help()} `and `{:reply,"You asked for \"#{content}\" but I have no answer! ...} `on line 25 and 40 to `:ephemeral` too, what do you think?

Reply:
perfect